Resources/Prototypes/Entities/Structures/Windows/window.yml

```diff
@@ -132,7 +134,7 @@
       mask:
       - FullTileMask
       layer:
-      - GlassLayer
+      - GlassAirlockLayer
```

Comment:
missed the # HIPPIE EDIT BEGINN thing here.

Reply:
yikes